test/platform/test-platform-execute.ts

```diff
@@ -2,74 +2,91 @@
 /// <reference types="@types/node" />
 import test from 'ava';
 import {
-  extractMockLocalHomePlatform,
-  DeviceManagerStub,
-  extractDeviceManagerStub,
-  MockLocalHomePlatform,
+  extractStubs,
+  UdpResponseData,
+  ERROR_EXECUTE_RESPONSE_ERROR_STATUS,
 } from '../../src';
+import {createIdentifyHandler} from './test-platform-fixtures';
 import {
+  createUdpDeviceCommand,
   createExecuteHandler,
-  createIdentifyHandler,
-  createDeviceCommand,
-  UdpResponseData,
-} from './test-platform-fixtures';
+} from '../example/fixtures';
 
 const DEVICE_ID = 'device-id-123';
 const DEVICE_PORT = 12345;
 const LOCAL_DEVICE_ID = 'local-device-id-123';
 const EXECUTE_REQUEST_ID = 'request-id-123';
 
+async function registerDevice(
+  app: smarthome.App,
+  executeHandler: smarthome.IntentFlow.ExecuteHandler,
+  deviceId: string,
+  localDeviceId: string
+): Promise<void> {
+  const identifyHandler = createIdentifyHandler(deviceId, localDeviceId);
+  await app.onIdentify(identifyHandler).onExecute(executeHandler).listen();
+  await extractStubs(app).mockLocalHomePlatform.triggerIdentify(
+    'identify-request-id',
+    Buffer.from('test-buffer'),
+    deviceId
+  ),
+    localDeviceId;
```

Comment:
Is this value indented correctly? I'm not sure I follow where this gets called.

Reply:
Huh, it actually doesn't belong there at all.  I removed it.  I'm surprised it compiled.